src/mqt/qecc/circuit_compilation/code_switching_compiler.py

```
@@ -0,0 +1,234 @@
+        """Add an edge of infinite capacity between two nodes.
+
+        Parameters
+        ----------
+        u : str
+            Source node identifier.
+        v : str
+            Target node identifier.
+        """
+        self.add_edge_with_capacity(u, v, capacity=float("inf"), bidirectional=bidirectional)
+
+    def add_regular_edge(self, u: str, v: str, capacity: float = 100.0, bidirectional: bool = True) -> None:
```

Ruff (0.14.3)

Boolean-typed positional argument in function definition

(FBT001)